notes/git-managing-remotes.md

```diff
@@ -0,0 +1,32 @@
+## MANAGING BRANCHES
+
+**1. Adding a remote repository**
+`git remote add` command on the terminal, in the directory your repository is stored at.
+
+The `git remote add` command takes two arguments:
+
+-   A remote name, i.e., `origin`
+-   A remote URL, i.e., `https://github.com/OWNER/REPOSITORY.git`
+
+**2. Changing a remote repository's URL**
+`git remote set-url` to change the the existing remte's URL. The command takes two arguments:
+
+-   An existing remote name. For example, `origin` or `upstream` are two common choices.
+-   A new URL for the remote.
+-   Use `git remote -v` to list remotes and check if the URL was updated.
+
+    **3. Renaming a remote repository**
```

Comment:
the is an space here in .3 that should not be.
<img width="562" alt="Screen Shot 2023-12-18 at 23 17 41" src="https://github.com/jacekroszkowiakdev/products-component/assets/42832783/5ac824b8-3ab4-414a-8b4e-23f50a727294">

Reply:
Corrected formatting in the whole file.